### ISSUE TITLE
Inconsistent attribute quoting in some error messages

### ISSUE THREAD

**Issue by OddBloke** [python/mypy#3887]
@ilevkivskyi has pointed out some inconsistent attribute quoting in their review of #3873. We've agreed to handle those in a separate PR, so I'm putting a list here so I don't lose track of them:

* `__aexit__` should be quoted in `Incompatible types in "async with" for __aexit__ (actual type "int", expected type "Awaitable[Any]")` 
(https://github.com/python/mypy/pull/3873/files/daa6792ae1ddaf85ce94dd8e674b27d119183509#r135941496)
* `await` should be quoted in `Incompatible types in await` (for consistency with `"yield from" can't be applied to ...`) (https://github.com/python/mypy/pull/3873/files/daa6792ae1ddaf85ce94dd8e674b27d119183509#r135941626)
* The first quoted part of `'Add5.__call__' has type "Callable[[Arg(int, 'x')], int]"` should use double-quotes (https://github.com/python/mypy/pull/3873/files/daa6792ae1ddaf85ce94dd8e674b27d119183509#r135942783)

**Comment by OddBloke** [comment]
@ilevkivskyi Can you confirm that I've captured your comments accurately?

(I apparently can't assign this to myself, but I will take care of it after #3873 has landed.)

**Comment by ilevkivskyi** [comment]
Yes, everything is right.

**Comment by OddBloke** [comment]
@ilevkivskyi I've addressed these in #3898.
